docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -473,14 +468,9 @@ You may also refer to the [secrets-config proxy](../../security/secrets-config-p
     * `privkey.pem` private key
     * `ca.pem` certificate authority file (if not available in root certificates)
 
-    Read the API Gateway token:
+    Replace the default certificate:
     ```bash
-    KONG_ADMIN_JWT=`sudo cat /var/snap/edgexfoundry/current/secrets/security-proxy-setup/kong-admin-jwt`
-    ```
-
-    Add the certificate, using Kong Admin JWT to authenticate:
-    ```bash
-    edgexfoundry.secrets-config proxy tls --incert cert.pem --inkey privkey.pem --admin_api_jwt $KONG_ADMIN_JWT
+    edgexfoundry.secrets-config proxy tls --incert cert.pem --inkey privkey.pem
```

Comment:
```suggestion
    sudo edgexfoundry.secrets-config proxy tls --incert cert.pem --inkey privkey.pem --targetfolder /var/snap/edgexfoundry/current/nginx
```
to solve `level=ERROR ts=2023-03-10T11:22:17.684834466Z app=secrets-config source=bootstraphandler.go:77 msg="open /etc/ssl/nginx/nginx.crt: no such file or directory"`.
and `level=ERROR ts=2023-03-10T11:27:55.119464373Z app=secrets-config source=bootstraphandler.go:77 msg="open /var/snap/edgexfoundry/current/nginx/nginx.crt: permission denied"`.

But then the snap isn't allowed to access to user files outside of the snap: `level=ERROR ts=2023-03-10T11:28:12.178784694Z app=secrets-config source=bootstraphandler.go:77 msg="open cert.pem: permission denied"`.

I had to move them to a writable snap directory (`/var/snap/edgexfoundry/current/temp`), to be presented with yet another error, which I could not solve:
```
sudo edgexfoundry.secrets-config proxy tls --incert /var/snap/edgexfoundry/current/temp/cert.pem --inkey /var/snap/edgexfoundry/current/temp/privkey.pem --targetfolder /var/snap/edgexfoundry/current/nginx
level=ERROR ts=2023-03-10T11:31:28.079184221Z app=secrets-config source=bootstraphandler.go:77 msg="chown /var/snap/edgexfoundry/current/nginx/nginx.crt: operation not permitted"
```
Why is [this](https://github.com/edgexfoundry/edgex-go/blob/97ead1fc80aa095b46845053832efa05950d1120/internal/security/config/command/proxy/tls/command.go#L122) trying to change the owner? The snap confinement disallows that:
```
$ snappy-debug
INFO: Following '/var/log/syslog'. If have dropped messages, use:
INFO: $ sudo journalctl --output=short --follow --all | sudo snappy-debug
= Seccomp =
Time: Mar 10 12:41:18
Log: auid=1000 uid=0 gid=0 ses=4 subj=snap.edgexfoundry.secrets-config pid=34685 comm="secrets-config" exe="/snap/edgexfoundry/4324/bin/secrets-config" sig=0 arch=c000003e 93(fchown) compat=0 ip=0x47949b code=0x50000
Syscall: fchown
Suggestions:
* don't copy ownership of files (eg, use 'cp -r --preserve=mode' instead of 'cp -a')
* try the snapcraft preload plugin: https://github.com/sergiusens/snapcraft-preload
* adjust program to not use 'fchown'
* ignore the denial if the program otherwise works correctly (unconditial chown is often just noise)
```

Reply:
Can an alternative be to directly replace `nginx.crt` and `nginx.key` and reload / restart nginx?
```
$ sudo tree /var/snap/edgexfoundry/current/nginx/
/var/snap/edgexfoundry/current/nginx/
├── conf.d
│   ├── edgex-custom-rewrites.inc
│   └── edgex-default.conf
├── nginx.conf
├── nginx.crt
└── nginx.key
```

Reply:
> Can an alternative be to directly replace `nginx.crt` and `nginx.key` and reload / restart nginx?

Certainly that sounds easier.  If it was just a single file I could take it from STDIN.